app/components/ui/molecules/StaffGrid.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components'
+import StaffCard from './StaffCard'
+
+const { Flex, Box } = require('grid-styled')
+
+const Grid = styled(Flex)`
+  flex-flow: row wrap;
+`
+const Card = styled(Box)`
+  padding-top: 20px;
+`
+
+const StaffGrid = ({ staff }) => (
+    <Grid>
+      {staff.map(person => (
+        <Card Box px={2} width={1 / 3}>
```

Comment:
Having padding on both sides causes the cards to be indented from the text. Have a look at how `TwoColumnLayout` uses negative margins to prevent this.

Also when resized, the images get squeezed. Fixing this probably requires changes to `StaffCard`.